tuf/repository_tool.py

```diff
@@ -2639,34 +2738,60 @@ def delegate_hashed_bins(self, list_of_targets, keys_of_hashed_bins,
       target = self._create_delegated_target(bin_role['name'], keyids, 1,
           relative_paths)
 
-      roleinfo = {'name': bin_role['name'],
-                  'keyids': keyids,
-                  'threshold': 1,
-                  'terminating': False,
-                  'path_hash_prefixes': bin_role['target_hash_prefixes']}
-      delegated_roleinfos.append(roleinfo)
-
       for key in keys_of_hashed_bins:
         target.add_verification_key(key)
 
+      if not succinct:
+        roleinfo = {'name': bin_role['name'],
+                    'keyids': keyids,
+                    'threshold': 1,
+                    'terminating': False,
+                    'path_hash_prefixes': bin_role['target_hash_prefixes']}
+        delegated_roleinfos.append(roleinfo)
+      else:
+        roleinfo = {'name': bin_role['name'],
+                    'keyids': keyids,
+                    'signing_keyids': [],
+                    'version':1,
+                    'threshold': 1,
+                    'terminating': False,
+                    'paths': []}
```

Comment:
Are an empty `paths` and the `version` required? If so we should add them for non-succinct hbd.

Reply:
In fact, we don't appear to be doing anything with this roleinfo dict? Let's just delete this else clause?